src/docfx/template/TemplateEngine.cs

```diff
@@ -60,6 +87,20 @@ internal class TemplateEngine
             _searchIndexBuilder = searchIndexBuilder;
         }
 
+        private TemplateEngine(ErrorBuilder errors, Config config, Package package, string locale)
```

Comment:
Is it possible to have only one private constructor to remove the duplicated logic?

Reply:
Done